v2/cli/term/project_page.go

```diff
@@ -96,7 +96,16 @@ func (p *projectPage) fill(project core.Project, events core.EventList) {
 func (p *projectPage) fillProjectInformation(project core.Project) {
 	// Fill the project information.
 	p.projectInfo.Clear()
-	p.projectInfo.SetText(fmt.Sprintf("[yellow]Project: [white]%s", project.ID))
+	p.projectInfo.SetText(
+		fmt.Sprintf(
+			"[yellow]Project: [white]%s\n"+
+				"[yellow]Description: [white]%s\n"+
+				"[yellow]Time Created: [white]%s",
+			project.ID,
+			project.Description,
+			project.Created.Local().Format("2006-01-02 15:04:05"),
```

Comment:
Let's create a helper function for formatting dates like this. This won't be the only place.

Also let's not use local time. All our dates are recorded in UTC and everywhere else I can think of, they're displayed in UTC as well.

Reply:
Ok, since this will likely be used in the event page too, should this function be placed somewhere in the `internal` folder?

Reply:
Let's put it int he `helpers.go` file in this package.

fwiw, "helpers" or "utils" is usually considered a "bad smell" that indicates missed opportunities to find a _better_ place for a piece of code.

But we're fairly early in this `brig term` effort. I think _both_ of us are going to be collaborating on this for a few weeks.

We'll deal with finding a better home for the helpers later.